UnitySDK/Assets/ML-Agents/Examples/Tennis/Scripts/HitWall.cs

```diff
@@ -17,109 +18,111 @@ void Start()
         m_AgentB = m_Area.agentB.GetComponent<TennisAgent>();
     }
 
-    void OnTriggerExit(Collider other)
+    private void Reset()
```

Comment:
nit: the `private` modifier is redundant in c#.

Reply:
removed.